api/src/main/java/com/codeforcommunity/exceptions/UserDeletedException.java

```diff
@@ -0,0 +1,25 @@
+package com.codeforcommunity.exceptions;
+
```

Comment:
Not important, this came from my other PR, will disappear after merging that PR.